src/foam/core/Enum.js

```diff
@@ -384,7 +384,7 @@ foam.CLASS({
         return n
       },
       expression: function(type) {
-        return type && foam.lookup(type).VALUES[0];
+        return type && ( foam.lookup(type).forOrdinal(0) || foam.lookup(type).VALUES[0] );
```

Comment:
Would this also work if you just used 'of' instead of 'foam.lookup(type)'?

Reply:
no, not really sure why but installAxioms is changing the order in the sort even though they all have the same priority..

Reply:
I mean replace foam.lookup(type) to be replaced with just this.of?